python/setup.py

```diff
@@ -23,7 +23,11 @@
 
 
 def config_cython():
-    """Try to configure cython and retyurn cython configuration"""
+    """Try to configure cython and return cython configuration"""
+    if os.name == 'nt':
+        print("WARNING: Cython is not supported on Windows, will compile without cython module")
+        return []
```

Comment:
cython work in windows fine.just need install correct version compiler.
The compiler version depends on the python version

Visual C++|CPython
-|-
14.0|3.5
10.0|3.3, 3.4
9.0|2.6, 2.7, 3.0, 3.1, 3.2

and 9.0 ms provides a separate download. not need vs.